### ISSUE TITLE
Drops page: promised unlockable item not avaliable

### ISSUE THREAD

**Issue by JustLuuuu** [kodadot/nft-gallery#7882]
New generative drop page on ahk is promising free merch, once I minted and chceked the NFT - there was no unlockable content.

Promising hidden content:
![image](https://github.com/kodadot/nft-gallery/assets/104380776/86406268-0fcc-461e-a655-2a57296b738a)
![image](https://github.com/kodadot/nft-gallery/assets/104380776/5e8680cd-86c7-494c-aaea-44ff36f2ea40)

My NFT without the unlockable content: 

https://github.com/kodadot/nft-gallery/assets/104380776/ece7f153-9519-4e57-b193-6771e5326ebb

Either add the unlockable content or hide the unlockable info from the drop page

**Comment by yangwao** [comment]
cc @prury still thing?

**Comment by prury** [comment]
> cc @prury still thing?

nope, we removed the section saying that drop has unlock able content
